src/cpu/cpu.cpp

```diff
@@ -3240,21 +3240,59 @@ void init_cpu_dosbox_settings(Section_prop& secprop)
 	pstring->Set_values({
 		"auto",
 #if C_DYNAMIC_X86 || C_DYNREC
-		        "dynamic",
+		"dynamic",
 #endif
-		        "normal", "simple"
+		"normal", "simple"
 	});
+
 	pstring->Set_help(
-	        "CPU core used in emulation ('auto' by default). 'auto' will switch to dynamic\n"
-	        "if available and appropriate.");
+	        "Type of CPU emulation core to use ('auto' by default).\n"
+	        "  auto:     'normal' core for real mode programs, 'dynamic' core for protected\n"
+	        "            mode programs (default). Most programs will run correctly with this\n"
+	        "            setting.\n"
+	        "  normal:   The DOS program is interpreted instruction by instruction. This\n"
+	        "            yields the most accurate timings, but puts 3-5 times more load on\n"
+	        "            the host CPU compared to the 'dynamic' core. Therefore, it's\n"
+	        "            generally only recommended for real mode programs that don't need\n"
+	        "            a fast emulated CPU or are timing-sensitive. The 'normal' core is\n"
+	        "            also necessary for programs that self-modify their code.\n"
+	        "  simple:   The 'normal' core optimised for old real mode programs; it might\n"
+	        "            give you slightly better compatibility with older games. Auto-\n"
+	        "            switches to the 'normal' core in protected mode.\n"
```

Comment:
Oh I see what you're saying after re-reading it.  If you set `simple`, it will change to `normal` when you enter protected mode.  I thought you were talking about the `auto` setting.

Nevermind this string is fine :smile: